cli/scan.js

```diff
@@ -0,0 +1,65 @@
+const assert = require('assert')
+const dotenv = require('dotenv')
+const { getSecurityVulnerabilitiesForMultiple } = require('../index')
+const { writeToCsv } = require('../src/csv')
+const { exitWithError } = require('../src/error')
+
+dotenv.config()
+
+const bbUsername = process.env.BITBUCKET_USERNAME
+const bbAppPassword = process.env.BITBUCKET_APP_PASSWORD
+const token = process.env.GITHUB_TOKEN
+
+const args = process.argv.splice(2)
+
+try {
+  assert(
+    process.argv.length > 2,
+    `Usage:\n  node cli/scan.js --github="org" --bitbucket="workspace" --csv="versions.url"\n\n` +
+    `  -g, --github       The Github organization to scan\n` +
+    `  -b, --bitbucket    The Bitbucket workspace to scan\n` +
+    `  -c, --csv          The url to the versions csv file\n`
+  )
+
+  assert(token, 'Please provide a GITHUB_TOKEN environment variable to check for security vulnerabilities')
+
+  const bitbucket = { }
+  const github = { token }
+  let versionsCsvUrl = ''
+
+  for (const arg of args) {
+    const [prefix, value] = arg.split('=', 2)
+    const data = value?.replace('"', '')
+
+    switch (prefix) {
+      case '-b', '--bitbucket':
+        bitbucket.workspace = data
+        break
```

Comment:
Just checking that you do want all these `break`s in here, so that you ignore subsequent args. (Or am I misunderstanding the code?)

Reply:
Isn't the `break` working on the for loop. So, when it gets the first arg, it breaks out of the for loop and stops checking for more args?

Reply:
Looks like I'm wrong about that, i.e. that the `break` is actually working with the `switch` and not with the for loop.